.swiftformat

```diff
@@ -0,0 +1,19 @@
+# file options
+
+# format options
+
+--commas inline
+--header ignore
+--indent 4
+--patternlet inline
+--operatorfunc no-space
```

Comment:
I'm not sure that this is correct rule.

As written in [Rules.md](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#spacearoundoperators) default value is `spaced`, so it means that statements like that:
```swift
a+b+c
```

Would be replaced by this:
```swift
a + b + c
```

But you have disabled this rule, haven't you? 🙂